src/content/docs/iast/summary.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: New Relic IAST summary page
+tags:
+    - New Relic IAST
+    - IAST
+metaDescription: Find out the data you can find on the New Relic IAST summary page.
+---
+
+The New Relic IAST summary page provides a centralized view of your tested applications and your vulnerabilities at a glance. View vulnerabilities and their risk scores helps you to prioritize remediation decisions.
+
+Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > IAST** to open the New Relic IAST summary page.
+
+<img
+    title="New Relic IAST Summary page"
+    alt="New Relic IAST Summary page"
+    src={iastSummary}
+/>
+
+<figcaption>
```

Comment:
There's a discrepancy between what's in the caption and what's in the path up above.